### ISSUE TITLE
seeing log-level of a message

### ISSUE THREAD

**Issue by eval** [harto/huon#3]
Currently you cannot see what the log-level of an individual message is.  
Would you be interested in a PR that prepends messages with the log-level?

Example:
current:
```
[1.880s] [foo.ns1] just some info
[2.880s] [foo.ns2] something going wrong here
```

wanted:
```
[1.880s] [foo.ns1] INFO just some info
[2.880s] [foo.ns2] ERROR something going wrong here
```

**Comment by harto** [comment]
Sure! Maybe we can follow this same pattern of putting metadata in square brackets, e.g.
```
[1.880s]  [foo.ns1]  [INFO]  something happened
```

**Comment by harto** [comment]
Fixed in #4. (Thanks!)
